libraries/provision/ansible/playbooks/files/telegraf-base.conf

```diff
@@ -0,0 +1,1577 @@
+# Telegraf Configuration
```

Comment:
For these configs, it may be cleaner to only include what settings we are using. What do you think @ajres?